main_test.go

```diff
@@ -544,6 +605,24 @@ func TestGetVMName(t *testing.T) {
 	}
 }
 
+func TestCheckIPv6Capable(t *testing.T) {
+	server := httptest.NewServer(nil)
+	defer server.Close()
+	overrideHTTP(server)
+	want := true
```

Comment:
Nit: move `want` to be declared right above where it is used.

Reply:
makes sense.. updating this